doozerlib/coverity.py

```diff
@@ -273,6 +273,9 @@ def _covscan_prepare_parent(cc: CoverityContext, parent_image_name, parent_tag)
 LABEL DOOZER_COVSCAN_GROUP_PARENT={cc.runtime.group_config.name}
 USER 0
 
+# Set https proxy
+ENV https_proxy http://proxy.util.phx1.redhat.com:8080
```

Comment:
Prefer not to hardcode a string in this location. Maybe query `os.environ` for the existence of `(https?|no)_proxy` vars and set them if defined?

Reply:
We may change to another proxy over time. Agree this shouldn't been hardcoded.

An option could be using `ARG` directive in Dockerfile then pass proxy related environment variables to the podman command line.

Reply:
We may change to another proxy over time. Agree this shouldn't been hardcoded.

An option could be using `ARG` directive in Dockerfile then pass proxy related environment variables to the podman command line.